src/Examples/Flowsheets/power_generation/rsofc/rsofc_soec_flowsheet.py

```diff
@@ -0,0 +1,2713 @@
+    GenericReactionParameterBlock,
+)
+
+from idaes.models_extra.power_generation.properties.natural_gas_PR import (
+    get_prop,
+    get_rxn,
+    EosType,
+)
+from idaes.models.properties import iapws95
+from CO2_H2O_Ideal_VLE import get_prop as CO2_H2O_VLE_config
+
+
+# Import costing
+import rsofc_costing as rsofc_cost
+from idaes.models_extra.power_generation.costing.power_plant_capcost import (
+    QGESSCosting,
+    QGESSCostingData,
+)
+
+# Import logger
+import idaes.logger as idaeslog
+
+
+def _set_port(port, F, T, P, comp, fix=True):
+    if fix:
+        port.flow_mol.fix(F)
+        port.temperature.fix(T)
+        port.pressure.fix(P)
+        for k, v in comp.items():
+            port.mole_frac_comp[:, k].fix(v)
+    else:
+        port.flow_mol[:].value = F
+        port.temperature[:].value = T
+        port.pressure[:] = P
+        for k, v in comp.items():
+            port.mole_frac_comp[:, k].value = v
+
+
+def add_flowsheet(m=None, name="SOEC Module"):
+    if not hasattr(m, "soec_fs"):
+        m.soec_fs = FlowsheetBlock(dynamic=False)
+        m.soec_fs.costing = QGESSCosting()
+    return m
+
+
+def add_properties(fs):
+    fuel_comp = {  # components present
+        "CH4",
+        "C2H6",
+        "C3H8",
+        "C4H10",
+        "O2",
+        "H2O",
+        "CO2",
+        "N2",
+        "Ar",
+    }
+    air_comp = {  # components present
+        "O2",
+        "H2O",
+        "CO2",
+        "N2",
+        "Ar",
+    }
+    fs.fg_prop = GenericParameterBlock(
+        **get_prop(components=fuel_comp, phases=["Vap"], eos=EosType.IDEAL)
+    )
+    fs.fg_prop.set_default_scaling("mole_frac_comp", 1e2)
+    fs.fg_prop.set_default_scaling("mole_frac_phase_comp", 1e2)
+    rxns = {  # reactions and key components for conversion
+        "ch4_cmb": "CH4",
+        "c2h6_cmb": "C2H6",
+        "c3h8_cmb": "C3H8",
+        "c4h10_cmb": "C4H10",
+    }
+    fs.rxns = rxns
+    fs.fg_combust = GenericReactionParameterBlock(**get_rxn(fs.fg_prop, rxns))
+    fs.water_prop = iapws95.Iapws95ParameterBlock()
+    fs.h2_prop = GenericParameterBlock(
+        **get_prop(components={"H2", "H2O"}, phases=["Vap"], eos=EosType.IDEAL)
+    )
+    fs.o2_prop = GenericParameterBlock(
+        **get_prop(components={"O2", "H2O"}, phases=["Vap"], eos=EosType.IDEAL)
+    )
+    fs.air_prop = GenericParameterBlock(
+        **get_prop(components=air_comp, phases=["Vap"], eos=EosType.IDEAL)
+    )
+    fs.air_prop.set_default_scaling("mole_frac_comp", 1e2)
+    fs.air_prop.set_default_scaling("mole_frac_phase_comp", 1e2)
+    fs.h2_compress_prop = GenericParameterBlock(
+        **get_prop(components={"H2"}, phases=["Vap"], eos=EosType.PR)
+    )
+    fs.h2_compress_prop.set_default_scaling("mole_frac_comp", 1e2)
+    fs.h2_compress_prop.set_default_scaling("mole_frac_phase_comp", 1e2)
+
+    fs.CO2_H2O_VLE = GenericParameterBlock(
+        **CO2_H2O_VLE_config(components=air_comp, phases=["Vap", "Liq"],)
```

Comment:
The CO2 H2O VLE property package was meant to be a temporary solution before the natural gas PR property package had support for liquid water. You should be able to replace `CO2_H2O_VLE_config` with `get_prop` here and delete the property package import and file.